src/scenes/ProgressReports/StatusHistory/StatusHistory.tsx

```diff
@@ -0,0 +1,114 @@
+import { useQuery } from '@apollo/client';
+import {
+  Box,
+  Breadcrumbs,
+  FormControlLabel,
+  Skeleton,
+  Switch,
+  Typography,
+} from '@mui/material';
+import { useToggle } from 'ahooks';
+import { Helmet } from 'react-helmet-async';
+import { flexColumn } from '~/common';
+import { Breadcrumb } from '~/components/Breadcrumb';
+import { useChangesetAwareIdFromUrl } from '~/components/Changeset';
+import { EngagementBreadcrumb } from '~/components/EngagementBreadcrumb';
+import { getReportLabel } from '~/components/PeriodicReports/reports';
+import { ProgressReportBreadcrumb } from '~/components/ProgressReportBreadcrumb';
+import { ProgressReportListBreadcrumb } from '~/components/ProgressReportListBreadcrumb';
+import { ProjectBreadcrumb } from '~/components/ProjectBreadcrumb';
+import { Navigate } from '~/components/Routing';
+import { Error } from '../../../components/Error';
+import {
+  ProgressReportDetailDocument,
+  ProgressReportDetailFragment,
+} from '../Detail/ProgressReportDetail.graphql';
+import { WorkFlowEventList } from '../Detail/WorkflowEventList';
+
+export const StatusHistory = () => {
+  const { id, changesetId } = useChangesetAwareIdFromUrl('reportId');
+  const [showNotes, setShowNotes] = useToggle(true);
+
+  const { data, error } = useQuery(ProgressReportDetailDocument, {
+    variables: {
+      id,
+      changesetId,
+    },
+  });
+  if (error) {
+    return (
+      <Error page error={error}>
+        {{
+          NotFound: 'Could not find quarterly report',
+          Default: "Error loading quarterly report's status history",
+        }}
+      </Error>
+    );
+  }
+
+  const report = data?.periodicReport as
+    | ProgressReportDetailFragment
+    | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- safety check. It's possible with manual user input.
```

Comment:
Maybe we don't make such a long comment and put the dev comment above the eslint comment. Something like this:
```suggestion
  //  safety check. It's possible with manual user input.
  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition 
 ```